src/vcpkg/commands.dependinfo.cpp

```diff
@@ -305,6 +305,7 @@ namespace vcpkg::Commands::DependInfo
             return check_and_get_full_package_spec(
                 std::string{arg}, default_triplet, COMMAND_STRUCTURE.example_text, paths);
         });
+        print_default_triplet_warning(args, args.command_arguments);
```

Comment:
I still think it would be better to do this as part of `check_and_get_full_package_spec` but that's nitpick territory.